homeassistant/components/hassio/__init__.py

```diff
@@ -144,7 +144,8 @@ async def async_install_addon(hass: HomeAssistantType, slug: str) -> None:
     """
     hassio = hass.data[DOMAIN]
     command = f"/addons/{slug}/install"
-    await hassio.send_command(command)
+    api_call = api_valid(hassio.send_command)
```

Comment:
Since `api_valid` parses the response and returns the right value, you can just use `api_valid` as a decorator on this function as long as you return the result from `await hassio.send_command()`

Reply:
Ok. It will change the return value of the function.